.gitignore

```diff
@@ -12,3 +12,6 @@ desktop.ini
 /bin/Debug
 /build
 /res/*.tis
+.idea
```

Comment:
```suggestion
/.idea
```

Reply:
@zrax Thank you very much for your feedback and for your first pass on the pull request. Unfortunately in these days I'm a bit busy with my work, but I'm going to tackle all the first pass comments as soon as I'm able to!